tests/roles/dataplane_adoption/tasks/nova_verify.yaml

```diff
@@ -23,8 +23,8 @@
   when: prelaunch_test_instance|bool
   ansible.builtin.shell: |
     {{ nova_header }}
-    ${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' && ${BASH_ALIASES[openstack]} server start test || echo PASS
-    ${BASH_ALIASES[openstack]} server list | grep -F '| test | ACTIVE |' && \
+    ${BASH_ALIASES[openstack]} server list --name test --status SHUTOFF && ${BASH_ALIASES[openstack]} server start test || echo PASS
+    ${BASH_ALIASES[openstack]} server list --name test --status ACTIVE && \
```

Comment:
need to double-check that the CLI returns rc > 0 if nothing found

Reply:
Unfortunately it doesn't, if we want non-zero RC then ``openstack server show test`` would be a better command.

$ openstack server show no-test
No Server found for no-test
$ echo $?
1

$ openstack server show test
------------------------------------------------------------
| Field                               | Value |
| OS-DCF:diskConfig                   | MANUAL |
...
| volumes_attached                    |  |
-------------------------------------------------------------

$ echo $?
0

Reply:
then could you please change the command so that it retains its original behavior (echoes PASS or not)

Reply:
Updated the commit, now using same server list command and matching Name and Status with grep for consistent behavior.